From 4e5af4465c74cbb659132def1b2c37b150e08171 Mon Sep 17 00:00:00 2001
From: Dong Aisheng <aisheng.dong@nxp.com>
Date: Thu, 7 Sep 2017 20:22:13 +0800
Subject: [PATCH 3335/5242] MLK-17491-41 clk: imx7ulp: do not keep GPIO clocks
 always on in clks_init_on

commit  ed7e58f9acd20bb6fb77a2a762c8a78f8e66da8c from
https://source.codeaurora.org/external/imx/linux-imx.git

Since patch ("gpio-vf610: add getting necessary clocks support"),
GPIO clocks will be handled by its driver.
No need put them in clks_init_on which should be only for system
critical clocks.

Fixes: ("1a86f07ce6a2 MLK-13485-4 clk: imx7ulp: add gpio port control clocks")
Acked-by: Peter Chen <peter.chen@nxp.com>
Reviewed-by: Fugang Duan <fugang.duan@nxp.com>
Signed-off-by: Dong Aisheng <aisheng.dong@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/imx/clk-imx7ulp.c |    5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/clk/imx/clk-imx7ulp.c b/drivers/clk/imx/clk-imx7ulp.c
index 54564af..6647b21 100644
--- a/drivers/clk/imx/clk-imx7ulp.c
+++ b/drivers/clk/imx/clk-imx7ulp.c
@@ -55,11 +55,6 @@
 	IMX7ULP_CLK_NIC1_DIV,
 	IMX7ULP_CLK_NIC1_BUS_DIV,
 	IMX7ULP_CLK_MMDC,
-	IMX7ULP_CLK_RGPIO2P1,
-	IMX7ULP_CLK_PCTLC,
-	IMX7ULP_CLK_PCTLD,
-	IMX7ULP_CLK_PCTLE,
-	IMX7ULP_CLK_PCTLF,
 };
 
 static void __init imx7ulp_clocks_init(struct device_node *scg_node)
-- 
1.7.9.5

